cmd/Makefile.am

```diff
@@ -128,14 +132,23 @@ noinst_PROGRAMS += decode-mount-opts/decode-mount-opts
 decode_mount_opts_decode_mount_opts_SOURCES = \
 	decode-mount-opts/decode-mount-opts.c
 decode_mount_opts_decode_mount_opts_LDADD = libsnap-confine-private.a
+decode_mount_opts_decode_mount_opts_STATIC =
+
+if STATIC_LIBCAP
+decode_mount_opts_decode_mount_opts_STATIC += -lcap
+else
+decode_mount_opts_decode_mount_opts_LDADD += -lcap
```

Comment:
I'm not sure I understand, did you mean to say `_STATIC` in one of the two rules above?